src/sentry/discover/compare_timeseries.py

```diff
@@ -21,11 +21,12 @@
 from sentry.search.eap.types import SearchResolverConfig
 from sentry.search.events.fields import get_function_alias
 from sentry.search.events.types import SnubaParams
+from sentry.snuba.dataset import Dataset
 from sentry.snuba.entity_subscription import apply_dataset_query_conditions
 from sentry.snuba.metrics import parse_mri_field
 from sentry.snuba.metrics.extraction import MetricSpecType
 from sentry.snuba.metrics_enhanced_performance import timeseries_query
-from sentry.snuba.models import SnubaQuery
+from sentry.snuba.models import ExtrapolationMode, SnubaQuery
```

Comment:
You need to import `from sentry_protos.snuba.v1.trace_item_attribute_pb2 import ExtrapolationMode`